packages/editor/src/client/locales/en.json

```diff
@@ -125,7 +125,7 @@
         "postImage": "Image for Post",
         "metadataInfo": "Information from these fields are being used for previews in Social Media Posts. If left blank, information from general-tab will be used instead.",
         "lightStyledWithNoImageError": "Light styled teasers must have an image",
-        "seoTitleHelpBlock": "Please use an SEO-optimised title"
+        "seoTitleHelpBlock": "Please use an SEO-optimised title. <1>SEO Guide</1>"
```

Comment:
what's the <1> tag?? :eyes:

Reply:
It allows you to write translations that also include tags like here an `<a>` tag. 

https://react.i18next.com/latest/trans-component